CanvasPlusPlayground/Model/Course.swift

```diff
@@ -9,120 +9,121 @@
  https://canvas.instructure.com/doc/api/courses.html
  A course object looks like:
  {
-   // the unique identifier for the course
```

Comment:
Could you somehow remove this from your staged changes please 👀 😿

Reply:
fixed!